README.md

```diff
@@ -1,10 +1,67 @@
-# Flexible Modeling System (FMS)
+# Modeling Framework: Flexible Modeling System (FMS)
+
+Today’s climate models simulate highly complex systems. In response to
+increasingly complex models, the climate community has developed tools and
+methodologies to facilitate the modeling process, developing modeling frameworks
```

Comment:
`, developing modeling frameworks
that provide infrastructure and tools to structure climate model codes and
facilitate many common tasks`--this sentence is redundant, and can be deleted